### PR TITLE
Add SecurityDataCollector if SecurityBundle is in the dependencies

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: WebProfilerServiceProvider.php [hunk 1]
```diff
@@ -70,19 +73,20 @@ public function register(Application $app)
...
-                'events'    => $app->share(function ($app) { return new EventDataCollector($app['dispatcher']); }),
-                'logger'    => $app->share(function ($app) { return new LoggerDataCollector($app['logger']); }),
-                'time'      => $app->share(function ($app) { return new TimeDataCollector(null, $app['stopwatch']); }),
-                'router'    => $app->share(function ($app) { return new RouterDataCollector(); }),
-                'memory'    => $app->share(function ($app) { return new MemoryDataCollector(); }),
+                'events' => $app->share(function ($app) { return new EventDataCollector($app['dispatcher']); }),
+                'logger' => $app->share(function ($app) { return new LoggerDataCollector($app['logger']); }),
+                'time' => $app->share(function ($app) { return new TimeDataCollector(null, $app['stopwatch']); }),
+                'router' => $app->share(function ($app) { return new RouterDataCollector(); }),
+                'memory' => $app->share(function ($app) { return new MemoryDataCollector(); }),
```
Comment (fabpot) [inline WebProfilerServiceProvider.php]: Changes in whitesapces should be reverted

**Comment by HeahDude** [comment]
Fixed and rebased.

**Comment by fabpot** [comment]
Didn't notice that it targets 1.0. Can you make one for master. We're about to release Silex 2.0, so 1.0 is frozen now. Thanks (the same probably goes for your other PRs).

**Comment by HeahDude** [comment]
Of courses, closing in favor of #78.

**Pull request closed without merging**